### ISSUE TITLE
Book.descriptionHTML not well formatted 

### ISSUE THREAD

**Issue by PFayoux** [FullStackBulletin/fullstack-books#1]
Hello,

I was using the API and I discovered that most of the book description contains a list at the end with items about what people should learn from the book. But all of the lists are not well formatted, there is only one `<li></li>` and other items start by ` - `.

It looks like the HTML should be fixed.

**Comment by PFayoux** [comment]
Ok so after investigation, we are parsing the description written in markdown here with the `marked` lib at [this place in the code](https://github.com/FullStackBulletin/fullstack-books/blob/main/scripts/build.ts#L70).

So for the moment I guess it's either a problem with the markdown or a problem with the lib (but I guess their lib is well written so let's check first what's on our side).

So we are getting our book [here](https://github.com/FullStackBulletin/fullstack-books/blob/main/scripts/build.ts#L42) from the file [book.yml](https://github.com/FullStackBulletin/fullstack-books/blob/main/src/books.yml).

We are parsing the yml file here using the `yaml`lib: https://github.com/FullStackBulletin/fullstack-books/blob/main/scripts/build.ts#L43

There might be an issue with the result of the parsing, I'll check that using the debuger.

Ok, so that's what I was expecting when parsing the file the description text is kind of put on a single line of text so we loose all the indentation usefull for markdown to be parsed. 

For example, this : 

```
JavaScript lies at the heart of almost every modern web application, from social apps like Twitter to browser-based game frameworks like Phaser and Babylon. Though simple for beginners to pick up and play with, JavaScript is a flexible, complex language that you can use to build full-scale applications.

    This much anticipated and thoroughly revised third edition of Eloquent JavaScript dives deep into the JavaScript language to show you how to write beautiful, effective code. It has been updated to reflect the current state of JavaScript and web browsers and includes brand-new material on features like class notation, arrow functions, iterators, async functions, template strings, and block scope. A host of new exercises have also been added to test your skills and keep you on track.

    As with previous editions, Haverbeke continues to teach through extensive examples and immerses you in code from the start, while exercises and full-chapter projects give you hands-on experience with writing your own programs. You start by learning the basic structure of the JavaScript language as well as control structures, functions, and data structures to help you write basic programs. Then you'll learn about error handling and bug fixing, modularity, and asynchronous programming before moving on to web browsers and how JavaScript is used to program them. As you build projects such as an artificial life simulation, a simple programming language, and a paint program, you'll learn how to:

    - Understand the essential elements of programming, including syntax, control, and data
    - Organize and clarify your code with object-oriented and functional programming techniques
    - Script the browser and make basic web applications
    - Use the DOM effectively to interact with browsers
    - Harness Node.js to build servers and utilities
    
    Isn't it time you became fluent in the language of the Web?
```

Is becoming that 😞 : 

```
‌JavaScript lies at the heart of almost every modern web application, from social apps like Twitter to browser-based game frameworks like Phaser and Babylon. Though simple for beginners to pick up and play with, JavaScript is a flexible, complex language that you can use to build full-scale applications.
This much anticipated and thoroughly revised third edition of Eloquent JavaScript dives deep into the JavaScript language to show you how to write beautiful, effective code. It has been updated to reflect the current state of JavaScript and web browsers and includes brand-new material on features like class notation, arrow functions, iterators, async functions, template strings, and block scope. A host of new exercises have also been added to test your skills and keep you on track.
As with previous editions, Haverbeke continues to teach through extensive examples and immerses you in code from the start, while exercises and full-chapter projects give you hands-on experience with writing your own programs. You start by learning the basic structure of the JavaScript language as well as control structures, functions, and data structures to help you write basic programs. Then you'll learn about error handling and bug fixing, modularity, and asynchronous programming before moving on to web browsers and how JavaScript is used to program them. As you build projects such as an artificial life simulation, a simple programming language, and a paint program, you'll learn how to:
- Understand the essential elements of programming, including syntax, control, and data - Organize and clarify your code with object-oriented and functional programming techniques - Script the browser and make basic web applications - Use the DOM effectively to interact with browsers - Harness Node.js to build servers and utilities
Isn't it time you became fluent in the language of the Web?
```

I'm gonna check the lib if we can do something about it.

**Comment by lmammino** [comment]
Thanks for this detailed investigation @PFayoux, let me know if you manage to do some additional progress. I won't have much time to spend on investigating this in the coming weeks, unfortunately. But if you can find a solution and submit a PR that would be awesome

**Comment by PFayoux** [comment]
Ok I found a way to fix it, basically we have to fix the markdown. I'm gonna submit a PR as soon as I finish it ;)

**Comment by PFayoux** [comment]
@lmammino I'm sorry if I didn't respected any expected format for the PR, I just did what seems good for me. I don't know if you expected something particular 😁

**Comment by lmammino** [comment]
Thanks a million for this contribution! Included in v1.2.0
